geometry_msgs/Pose distanceToGo
---
string action